risc0/zkvm/circuit/constants.h

```diff
@@ -74,4 +75,6 @@ CONSTSCALAR size_t kGlobalSize = kAccumMixGlobalOffset + kAccumMixGlobalSize;
 
 CONSTSCALAR size_t kComboCount = 5;
 
+CONSTSCALAR size_t kMaxCodeDigestCount = log2Ceil(kMaxCycles / kMinCycles) + 1;
```

Comment:
Can we document what this constant means?  Is it related to the "limit" referenced other places in this PR?